.swiftlint.yml

```diff
@@ -1,8 +1,10 @@
 disabled_rules:
-  - line_length
+  - cyclomatic_complexity
   - identifier_name
-  - trailing_whitespace
+  - line_length
   - nesting
+  - trailing_whitespace
+  - type_body_length
```

Comment:
It was complaining that `EventCreatingTests` and `EventDecodingTests` exceeded 250 lines. Not sure if we care that much about it, but it's possible that we could split the file up.

Reply:
These are fine to opt out.